rollbar-spring-webmvc/src/main/java/com/rollbar/spring/webmvc/RollbarExceptionResolver.java

```diff
@@ -0,0 +1,35 @@
+package com.rollbar.spring.webmvc;
+
+import com.rollbar.notifier.Rollbar;
+import com.rollbar.notifier.config.ConfigBuilder;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+import org.springframework.web.servlet.ModelAndView;
+
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class RollbarExceptionResolver implements HandlerExceptionResolver {
+
+  private Rollbar rollbar;
+
+  public RollbarExceptionResolver(Rollbar rollbar) {
+    this.rollbar = rollbar;
+  }
+
+  public RollbarExceptionResolver(ConfigBuilder configBuilder) {
```

Comment:
I think we should remove this constructor and use the one that gets rollbar as dependency.

Reply:
done